core/new-gui/src/app/workspace/component/navigation/navigation.component.scss

```diff
@@ -7,10 +7,17 @@
   z-index: 1;
 }
 
+
+.user-icon-navigation-page{
+  position:relative;
+  top:-3px;
+  left:890px;
```

Comment:
do not use absolute pixel value, this should scale as user's browser size changes.